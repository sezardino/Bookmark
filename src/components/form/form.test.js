import React from 'react';
import {shallow} from 'enzyme';

import Form from './form';

describe('Component Form', () => {
	const component = shallow(<Form />);

	it('Component Form rendered correctly', () => {
		expect(component.debug()).toMatchSnapshot();
	});
});
