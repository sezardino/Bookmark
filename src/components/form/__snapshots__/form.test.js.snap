// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Form Component Form rendered correctly 1`] = `
"<form action=\\"#\\" method=\\"post\\" className=\\"form subscribe__form \\" onSubmit={[Function: onSubmit]}>
  <label htmlFor=\\"mail\\" className=\\"form__error\\">
    <input type=\\"email\\" id=\\"mail\\" name=\\"mail\\" minLength={3} placeholder=\\"Enter your email address\\" className=\\"form__mail\\" value=\\"\\" onChange={[Function: onChange]} />
    <p className=\\"img-wrapper form__img\\">
      <img src=\\"./img/icon-error.svg\\" alt=\\"error\\" />
    </p>
    <span className=\\"form__indicator\\" />
  </label>
  <button type=\\"submit\\" className=\\"button button--red button--form form__button\\" onClick={[Function: submitHandler]}>
    Contact us
  </button>
</form>"
`;
